pyproject.toml

```diff
@@ -197,6 +190,13 @@ classmethod-decorators = [
 known-first-party = ["polyfactory", "tests"]
 
 [tool.ruff.per-file-ignores]
+"**/*.*" = ["ANN101", "ANN401", "ANN102", "TD",]
+"polyfactory/**/*.*" = ["PLR0913",
+    "ARG005", # Investigate @guacs
```

Comment:
@guacs there are many investigate tags for you to check out that I am unsure about